package.json

```diff
@@ -22,7 +22,13 @@
     "@angular/platform-browser-dynamic": "~8.2.13",
     "@angular/router": "~8.2.13",
     "@ng-bootstrap/ng-bootstrap": "^5.1.4",
+    "angular-bootstrap-md": "^8.6.0",
+    "bootstrap": "^4.3.1",
+    "canvasjs": "^1.8.1",
+    "chart.js": "^2.9.3",
     "hammerjs": "^2.0.8",
+    "jquery": "^3.4.1",
```

Comment:
@YeetGrandpa Kannst du mir sagen was ^ und ~ in den Versionsnummern bedeutet?

Reply:
Das "^" bedeutet das auch neuere versionen verwendet werden dürfen und "~" beudetet das neuere versionen verwendet werden dürfen, wenn sie bugfixes enthalten.
https://bytearcher.com/articles/semver-explained-why-theres-a-caret-in-my-package-json/